protocol/contracts/lease/src/contract/endpoins.rs

```diff
@@ -17,10 +17,14 @@ use crate::{
     error::ContractResult,
 };
 
-use super::state::{self, Response, State};
+use super::state::{
+    self,
+    v8::{Migrate, State as State_v8},
+    Response, State,
+};
 
-// const CONTRACT_STORAGE_VERSION_FROM: VersionSegment = 7;
-const CONTRACT_STORAGE_VERSION: VersionSegment = 8;
+const CONTRACT_STORAGE_VERSION_FROM: VersionSegment = 8;
```

Comment:
Maybe introducing a `migration` feature like in admin contract and others could prove beneficial here. This won't require us to keep commented code. :)
```suggestion
#[cfg(feature = "migration")]
const CONTRACT_STORAGE_VERSION_FROM: VersionSegment = 8;
```